test/e2e/simple_test.go

```diff
@@ -69,7 +69,7 @@ func TestE2ESimple(t *testing.T) {
 
 	totalTxs := 0
 	for _, block := range blockchain {
-		require.Equal(t, v1.Version, block.Version.App)
+		require.Equal(t, appconsts.LatestVersion, block.Version.App)
```

Comment:
[no change needed] we can probably close https://github.com/celestiaorg/celestia-app/pull/3201 if this PR merges